pkg/acceptance/bettertestspoc/config/resource_model_test.go

```diff
@@ -0,0 +1,138 @@
+package config_test
```

Comment:
I would rename this file because `_test` indicates that some tests are here.

Reply:
`_test` suffix is a special suffix allowing you to have two different packages in one directory: one production and one for used only for tests (not necessarily the tests themselves but e.g. test structs too - like here)